Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -155,6 +155,7 @@ void UpdateFont()
 		void UpdateKeyboard()
 		{
 			Control.ApplyKeyboard(Element.Keyboard);
+			Control.ReloadInputViews();
```

Comment:
`Entry` and `Editor` derive from `InputView` which has a `Keyboard` property whereas `SearchBar` is not an `InputView`.